cves/CVE-2010-2068.yml

```diff
@@ -3,36 +3,38 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE:
+CWE: CWE-200
 CVSS: AV:N/AC:L/Au:N/C:P/I:N/A:N
 curated_instructions: |
   If you are manually editing this file, then you are "curating" it. Set the
   entry below to "true" as soon as you start. This will enable additional
   integrity checks on this file to make sure you fill everything out properly.
   If you are a student, we cannot accept your work as finished unless curated is
   set to true.
-curated: false
+curated: true
 announced_instructions: |
   Was there a date that this vulnerability was announced to the world? You can
   find this in changelogs, blogs, bug reports, or perhaps the CVE date. A good
   source for this is Chrome's Stable Release Channel
   (https://chromereleases.googleblog.com/).
   Please enter your date in YYYY-MM-DD format.
-announced: 2010-06-18T16:30Z
+announced: 2010-05-25
 description_instructions: |
   You can get an initial description from the CVE entry on cve.mitre.org. These
   descriptions are a fine start, but they can be kind of jargony.
-
   Rewrite this description in your own words. Make it interesting and easy to
   read to anyone with some programming experience. We can always pull up the NVD
   description later to get more technical.
-
   Try to still be specific in your description, but remove Chromium-specific
   stuff. Remove references to versions, specific filenames, and other jargon
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
```

Comment:
The description doesn't do much to paint a picture as to what the vulnerability is for the reader.  Maybe talk a little bit about the "timeout conditions" involved, or the "worker pools"

Reply:
Thank you! I have updated the description to be more detailed with information about the specific timeout condition as well as how the pool is involved.